src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -63,11 +63,7 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
         public static async Task<AuthorizationLevel> DetermineAuthorizationLevelAsync(HttpRequestMessage request, FunctionDescriptor function, IDependencyResolver resolver)
         {
             var secretManager = resolver.GetService<ISecretManager>();
-            var settings = resolver.GetService<WebHostSettings>();
-
-            var authorizationLevel = settings.IsAuthDisabled
-                ? AuthorizationLevel.Admin
```

Comment:
We were doing this inconsistently. Here in the functions controller, we translated disabled auth into Admin. However we weren't doing that for other controllers. I think its less confusing to just not do this transform.